### DISCUSSION TITLE
New update list compability games file pdf/tabel

### DISCUSSION THREAD

**Discussion by masterpiece67** [skyline-emu/skyline#1439] [Q&A] [answered]
hello dev, can you provide a pdf/table file regarding the latest updates to games that are already playable. thanks

**Comment by Crytonics** [comment, marked as answer]
We have on github https://github.com/skyline-emu/skyline-games-list/issues
You can sort by status(ingame, boot, nothing, playable), just click on "Labels" and then you can choose.
